### ISSUE TITLE
Pack batchNormalization

### ISSUE THREAD

**Issue by annxingyuan** [tensorflow/tfjs#774]
Batch normalization may be a good candidate for packing because it’s relatively simple and it shows up frequently.

By packing batchNormalization, we will prototype how to lazily pack / unpack textures when two packed ops are adjacent in the graph
